src/edu/ucsb/nceas/metacat/admin/PropertiesAdmin.java

```diff
@@ -52,381 +12,460 @@
+                        + "properties page: " + mue.getMessage());
+            } catch (GeneralPropertyException gpe) {
+                throw new AdminException("PropertiesAdmin.configureProperties - problem with properties while "
+                        + "processing system properties page: " + gpe.getMessage());
+            }
+        }
+    }
+
+
+    /**
+     * In the web.xml of the Metacat-index context, there is a parameter:
+     * <context-param>
      * <param-name>metacat.properties.path</param-name>
      * <param-value>/metacat/WEB-INF/metacat.properties</param-value>
-     * <description>The metacat.properties file for sibling metacat deployment. Note that the context can change</description>
+     * <description>The metacat.properties file for sibling metacat deployment. Note that the
+     * context can change</description>
      *  </context-param>
-     *  It points to the default metacat context - knb. If we rename the context, we need to change the value of there.
-	 */
-	private void modifyIndexContextParams(String indexContext) {
-	    if(indexContext != null) {
-	        try {
-	            String metacatContext = PropertyService.getProperty("application.context");
-	            //System.out.println("the metacat context is ========================="+metacatContext);
-	            if(metacatContext != null && !metacatContext.equals(DEFAULTMETACATCONTEXT)) {
-	                String indexConfigFile = 
-	                                PropertyService.getProperty("application.deployDir")
-	                                + FileUtil.getFS()
-	                                + indexContext
-	                                + FileUtil.getFS() 
-	                                + "WEB-INF"
-	                                + FileUtil.getFS()
-	                                + "web.xml";
-	                //System.out.println("============================== the web.xml file is "+indexConfigFile);
-	                String configContents = FileUtil.readFileToString(indexConfigFile, "UTF-8");
-	                //System.out.println("============================== the content of web.xml file is "+configContents);
-	                configContents = configContents.replace(BACKSLASH+DEFAULTMETACATCONTEXT+METACATPROPERTYAPPENDIX, BACKSLASH+metacatContext+METACATPROPERTYAPPENDIX);
-	                FileUtil.writeFile(indexConfigFile, new StringReader(configContents), "UTF-8");
-	            }
-                
-            } catch (Exception e) {
-                String errorMessage = "PropertiesAdmin.configureProperties - Problem getting/setting the \"metacat.properties.path\" in the web.xml of the index context : " + e.getMessage();
-                logMetacat.error(errorMessage);
+     *  It points to the default metacat context - knb. If we rename the context, we need to
+     *  change the value of there.
+     *
+     *  There is also a parameter:
+     * <context-param>
+     * <param-name>site.properties.path</param-name>
```

Comment:
This is no longer needed, now that we're persisting the site properties location in the metacat.properties file. Will refactor